samples/matter/thermostat/README.rst

```diff
@@ -272,17 +275,14 @@ After building this sample and the :ref:`Matter weather station <matter_weather_
 
    .. code-block:: console
 
-      $ ./chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, \
-      {"fabricIndex": 1, "privilege": 1, "authMode": 2, "subjects": [<thermostat_node_ID>], "targets": \
-      [{"cluster": 1026, "endpoint": <thermostat_sensor_endpoint_ID>, "deviceType": null}]}]' <Sensor Device NodeId> 0
+      $ ./chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, {"fabricIndex": 1, "privilege": 1, "authMode": 2, "subjects": [<thermostat_node_ID>], "targets": [{"cluster": 1026, "endpoint": <temperature_sensor_endpoint_ID>, "deviceType": null}]}]' <temperature_sensor_node_ID> 0
 
 #. Write a binding table to the thermostat to inform the device about the temperature sensor endpoint.
-   Use the following command, with *<temperature_sensor_node_ID>* and *<thermostat_sensor_endpoint_ID>* values from the previous step about commissioning:
+   Use the following command, with *<temperature_sensor_node_ID>*, *<thermostat_sensor_endpoint_ID>* and *<temperature_sensor_node_ID>* values from the previous step about commissioning:
```

Comment:
Please read this bullet point and the above one again to make sure the parameter names match the ones used in the command. For example, here you mention two times the same parameter `temperature_sensor_node_ID` and `thermostat_sensor_endpoint_ID` which is not present in the command. On the other hand we have in command `temperature_sensor_endpoint_ID` and `temperature_sensor_endpoint_ID` that are not described in the description.

For the bullet point above (regarding accesscontrol write acl) the `temperature_sensor_node_ID` is missing in the description.